packages/ketcher-polymer-editor-react/src/Editor.tsx

```diff
@@ -162,11 +162,6 @@ function Editor({ theme }: EditorProps) {
 
   const handleOpenPreview = useCallback(
     (e) => {
-      const tools = ['erase', 'select-rectangle', 'bond-single'];
```

Comment:
Are you sure that this should be removed? There is nothing about this logic in initial task

Reply:
I have removed it from PR. Thank you.